parser/ast/dml.go

```diff
@@ -1938,6 +1952,23 @@ func (n *LoadDataStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+type LoadDataOpt struct {
+	Name  string
+	Value ExprNode
```

Comment:
What will happen if `Value` contains subquery?

Reply:
only literal is allowed, see syntax